src/mmw/apps/home/views.py

```diff
@@ -29,5 +30,22 @@ def home_page(request):
     return render_to_response('home/home.html', csrf_token)
 
 
+def model(request, proj_id=None):
+    """
+    If proj_id was specified, check that the user owns
+    the project. It not, return a 404. Otherwise, just
+    load the index template and the let the front-end
+    handle the route and request the project through
+    the API.
+    """
+    csrf_token = {}
+    csrf_token.update(csrf(request))
```

Comment:
Do we really need to pass the csrf token to the client like this? We attach the csrf token as a header to each ajax call now.